action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
   folder:
     description: 'The folder which is deployed to Contentful App Hosting'
     required: true
+  allow-tracking:
+    description: 'The flag to allow tracking of the app-definition-id, branch deployed'
```

Comment:
I would not be that specific here.

"Flag that allows us to track the usage of the GitHub action"

Otherwise, we need to change this every time we adjust the metrics.